internal/pkg/cli/cmd/sync/init_test.go

```diff
@@ -1,118 +0,0 @@
-package sync_test
-
-import (
-	"strings"
-	"sync"
-	"testing"
-	"time"
-
-	"github.com/Netflix/go-expect"
-	"github.com/stretchr/testify/assert"
-
-	"github.com/keboola/keboola-as-code/internal/pkg/env"
-	"github.com/keboola/keboola-as-code/internal/pkg/testcli"
-	"github.com/keboola/keboola-as-code/internal/pkg/testhelper"
-	"github.com/keboola/keboola-as-code/internal/pkg/testproject"
-)
-
-func TestMissingParams(t *testing.T) {
```

Comment:
3: presunute do funkcionalneho testu `init-missing-params`, tam to dava vacsi zmysel.